service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -101,8 +101,10 @@ public SendingAmqpLink SendingLink
 
         public async override Task CloseAsync()
```

Comment:
>CloseAsync() [](start = 35, length = 12)

I would remove both changes to this method. The dispose method will take care of all disposing actions.